apps/server/src/modules/oauth-provider/service/oauth-provider.login-flow.service.spec.ts

```diff
@@ -68,12 +96,7 @@ describe('OauthProviderLoginFlowService', () => {
 				skip: true,
 				subject: 'subject',
 			} as ProviderLoginResponse;
```

Comment:
as ProviderLoginResponse should not needed if ProviderLoginResponse is defined as target.

Reply:
Done for the whole object. For the partial responses we need this.